dubbo-remoting/dubbo-remoting-netty4/src/main/java/org/apache/dubbo/remoting/transport/netty4/NettyConfigOperator.java

```diff
@@ -36,6 +37,8 @@ public class NettyConfigOperator implements ChannelOperator {
     private final Channel channel;
     private ChannelHandler handler;
 
+    private ProtocolDetector.Result detectResult;
```

Comment:
Where will this be used? I don't seem to see a place to use it?

Reply:
This is for the `triple` protocol to support both `http1` and  `http2`